test-engineering/contract-tests/client/tests/kinto.py

```diff
@@ -0,0 +1,98 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Set
+
+import requests
+from pydantic import BaseModel
+from requests import Response as RequestsResponse
+
+
+class KintoEnvironment(BaseModel):
+    """Class that holds information about Kinto environment variables."""
+
+    api: str
```

Comment:
nit: do you mind renaming this field for consistency? Merino uses `server` in the configuration for the server URL.

Reply:
change applied: https://github.com/mozilla-services/merino/pull/409/commits/670814d39d81946a96d7d0e48d99c0e89adce4fd

Agreed this is better.